project/plugins.sbt

```diff
@@ -7,11 +7,14 @@ resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.1.1"))
 
+addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.8.0-RC1"))
```

Comment:
Unfortunately, this creates a circular dependency with Play, something that already exists for Twirl:

https://github.com/playframework/twirl/blob/b712905b2ff8877b9fcf5fec8ec8f247a1ec5ea6/docs/project/plugins.sbt#L7

A tolerable problem though, since this does not needs to be up to date with Play releases.